docs/going-to-production.md

```diff
@@ -128,7 +128,7 @@ You can also log and track exceptions with a tool like Sentry. [This example](ht
 
 To improve loading performance, you first need to determine what to measure and how to measure it. [Core Web Vitals](https://vercel.com/blog/core-web-vitals?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) is a good industry standard that is measured using your own web browser. If you are not familiar with the metrics of Core Web Vitals, review this [blog post](https://vercel.com/blog/core-web-vitals?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) and determine which specific metric/s will be your drivers for loading performance. Ideally, you would want to measure the loading performance in the following environments:
 
-- In the lab, using your own computer or a simulator.
+- In the lab, use your own computer or a simulator.
```

Comment:
```suggestion
- In the lab, using your own computer or a simulator.
```

Would not align with the following line, but based on the context, I think "using" is more correct here anyway.